### PR TITLE
Adjust scaling

### PR DESCRIPTION
**Pull request by pega-roska**
Adjusting default scaling behavior based on performance feedback

### CODE REVIEW

**Comment by pegaautomationuser** [comment]
Starting PR-275 validation on -> integ-all

**Comment by pegaautomationuser** [comment]
Starting PR-275 validation on -> integ-all

**Comment by pegaautomationuser** [comment]
Starting PR-275 validation on -> integ-all

**Comment by pegaautomationuser** [comment]
Starting PR-275 validation on -> integ-all

File: charts/pega/README.md [hunk 1]
```diff
@@ -435,10 +435,11 @@ Parameter           | Description    | Default value
 ---                 | ---       | ---
 `hpa.minReplicas`   | Minimum number of replicas that HPA can scale-down | `1` 
 `hpa.maxReplicas`   | Maximum number of replicas that HPA can scale-up  | `5`
-`hpa.targetAverageCPUUtilization` | Threshold value for scaling based on initial CPU request utilization (The default value is `70` which corresponds to 70% of 2) | `70`
-`hpa.targetAverageMemoryUtilization` | Threshold value for scaling based on initial memory utilization (The default value is `85` which corresponds to 85% of 6Gi ) | `85`
+`hpa.targetAverageCPUValue` | Threshold value for scaling based on absolute CPU usage | `2.8`
```
Comment (taz-pega-work) [inline charts/pega/README.md:438]: Can you please add what 2.8 represents? A context would be useful to lead clients to make decisions and not guess.

**Comment by pegaautomationuser** [comment]
Starting PR-275 validation on -> integ-all

**Comment by pegaautomationuser** [comment]
Starting PR-275 validation on -> integ-all

File: charts/pega/README.md [hunk 2]
```diff
@@ -435,10 +435,11 @@ Parameter           | Description    | Default value
...
+`hpa.targetAverageCPUUtilization` | Threshold value for scaling based on initial CPU request utilization (Can be set instead of `hpa.targetAverageCPUValue` to set the threshold as a percentage of the requested CPU) | 
+`hpa.targetAverageMemoryUtilization` | Threshold value for scaling based on initial memory utilization (The default value is `85` which corresponds to 85% of 12Gi ) | `85`
 `hpa.enableCpuTarget` | Set to true if you want to enable scaling based on CPU utilization or false if you want to disable it | true
-`hpa.enableMemoryTarget` | Set to true if you want to enable scaling based on memory utilization or false if you want to disable it  | true
+`hpa.enableMemoryTarget` | Set to true if you want to enable scaling based on memory utilization or false if you want to disable it (It is recommended to leave this disabled) | false
```
Comment (taz-pega-work) [inline charts/pega/README.md:442]: Pega recommends leaving this disabled

File: terratest/src/test/pega/data/values_hpa_disabletarget.yaml [hunk 3]
```diff
@@ -10,4 +10,4 @@ global:
       hpa:
         enabled: true
         # enableCpuTarget should be true as default
-        enableMemoryTarget: false
+        # enableMemoryTarget should be false as default
```
Comment (taz-pega-work) [inline terratest/src/test/pega/data/values_hpa_disabletarget.yaml:13]: 1. why is this editable if it should always be false?
2. move to line 13, before the parameter
3. change to->Ensure that enableMemoryTarget remains set to false.

**Review by taz-pega-work — APPROVED** [review]
A few minor suggestions

**Comment by pegaautomationuser** [comment]
Starting PR-275 validation on -> integ-all

**Comment by pegaautomationuser** [comment]
Starting PR-275 validation on -> integ-all

File: terratest/src/test/pega/data/values_hpa_disabletarget.yaml (hunk 3 above)
Comment (pega-roska) [reply terratest/src/test/pega/data/values_hpa_disabletarget.yaml:13]: This file is not shipped, it's just a test asset.

File: charts/pega/README.md (hunk 2 above)
Comment (pega-roska) [reply charts/pega/README.md]: done

File: charts/pega/README.md (hunk 1 above)
Comment (pega-roska) [reply charts/pega/README.md]: How about now?  Updated to include a basic description.

**Review by pega-talba — APPROVED** [review]

File: charts/pega/README.md (hunk 1 above)
Comment (taz-pega-work) [reply charts/pega/README.md]: can you cite an article that tells me (or anyone that is in a position to want to begin scaling), like a test engineer, how I can see 2.8? Previously they explained that 85 meant 85% of 6Gi. Can you give a hint what 2.8 is? I'm 97.2% sure that you didn't just make 2.8 up. ;-)

Comment (pega-roska) [reply charts/pega/README.md]: You should check the most recent version of the file, the thing shown in this conversation doesn't update.

**Review by APegaDavis — APPROVED** [review]

**Pull request merged**